Cargo.toml

```diff
@@ -26,9 +26,9 @@ missing_docs = "warn"
 [patch.crates-io]
 
 # Uncomment to use unreleased versions of hugr
-#hugr-core = { git = "https://github.com/CQCL/hugr.git" }
-#hugr = { git = "https://github.com/CQCL/hugr.git" }
-#hugr-cli = { git = "https://github.com/CQCL/hugr.git" }
+hugr-core = { git = "https://github.com/CQCL/hugr.git" }
+hugr = { git = "https://github.com/CQCL/hugr.git" }
+hugr-cli = { git = "https://github.com/CQCL/hugr.git" }
```

Comment:
Add an issue to remove this override once hugr 0.14 gets released.

Reply:
https://github.com/CQCL/tket2/issues/707